Blocks/src/Mapping/MappingSchema.cs

```diff
@@ -204,12 +204,20 @@ public object EnumToValue(Enum value)
 		/// </summary>
 		/// <param name="from">Type to convert from.</param>
 		/// <param name="to">Type to convert to.</param>
-		/// <returns>Convert expression.</returns>
-		// ReSharper disable once VirtualMemberNeverOverridden.Global
-		protected internal virtual LambdaExpression TryGetConvertExpression(Type from, Type to)
+		/// <param name="expr">Expression that converts a value of type <i>TFrom</i> to <i>TTo</i> or null.</param>
+		/// <returns><see langword="true" /> for successful expression, <see langword="false" /> if failed..</returns>
+		protected internal bool TryGetConvertExpression(Type from, Type to, out LambdaExpression expr)
```

Comment:
CJ uses return null-as-none pattern in multiple places, not only in ConvertBuilder.
As for me there's no profit with converting these into Try(… out var).

Vote to decline.

Reply:
Returning null is not good idea and not suggested in most coding conventions.
Modern C# has out var which makes usage of TryGet functions very easy and safe.

Reply:
I suggest to call these methods with OrDefault suffix at least.

Reply:
Let's continue [here](http://rsdn.org/forum/prj.codejam/7281633) (rus).

Reply:
What is our conclusion ? 
https://rsdn.org/forum/prj.codejam/7300161.1